### PR TITLE
feature/sflow

### PR DESCRIPTION
**Pull request by jrouzierinverse**
Description
===========
The library for parsing sFlow

Checklist
=========
- [ ] Document the feature
- [x] Add unit tests
- [ ] Add acceptance tests (TestLink)

### CODE REVIEW

**Comment by jrouzierinverse** [comment]
Please review

**Comment by julsemaan** [comment]
We need to add a rule in iptables.conf to allow the sflow traffic

**Comment by extrafu** [comment]
@julsemaan  Review and merge if all is fine. Don't forget to update the NEWS file.

File: go/sflow/doc/sflow_host.txt [hunk 1]
```diff
@@ -0,0 +1,731 @@
+
```
Comment (julsemaan) [inline go/sflow/doc/sflow_host.txt]: I don't think we really need to put this file in our repo

File: go/sflow/doc/sflow_version_5.txt [hunk 2]
```diff
@@ -0,0 +1,2803 @@
+
```
Comment (julsemaan) [inline go/sflow/doc/sflow_version_5.txt]: This file doesn't really need to be in our repo

Comment (jrouzierinverse) [reply go/sflow/doc/sflow_version_5.txt]: Done

**Pull request merged**